scripts/update/Updater.php

```diff
@@ -392,5 +397,28 @@ public function update($initialVersion){
         }
 
         $this->skip('16.0.0', '19.3.1');
+
+        if ($this->isVersion('19.3.1')) {
+            $importerConfig = [
+                MetadataImporter::INJECTOR_KEY => [\oat\taoQtiItem\model\qti\metadata\ontology\LomInjector::class],
```

Comment:
You can import `\oat\taoQtiItem\model\qti\metadata\ontology\LomInjector` for readability